src/osyris/core/vector.py

```diff
@@ -251,3 +254,26 @@ def cross(self, other):
         z = self.x * other.y
         z -= self.y * other.x
         return self.__class__(x, y, z)
+
+    @property
+    def r(self):
```

Comment:
I don't know if we need to think about a cylindrical radius also? Maybe `rc` or `rcyl`. Don't know what the best name is.

Reply:
yeah I have `r_cyl` in my unpushed changes. `phi` is the same as the spherical counterpart, and `z` is just the cartesian component. So all that's needed is a wrapper for `r_cyl` that computes `sqrt(x**2 + y**2)`.